about.html

```diff
@@ -0,0 +1,172 @@
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campaign">CC Campaign</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- HEADER ENDS -->
+
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-page about-page container">
+        <p class="greet about-hide">"Lead the world through innovations."</p>
+        <h1 class="about-header">
+          Connecting Minds, <br />
+          Empowering Change 2023
+        </h1>
+        <p class="description about-description">
+          "Welcome to the Innovation and Leadership Conference, a premier event
+          dedicated to fostering creativity, driving innovation, and promoting
+          effective leadership in today's dynamic and fast-paced business
+          environment."
+        </p>
+        <p class="about-location">
+          Please contact us via email for any further question and
+          clarifications about the ILC Summit 2023
+        </p>
+        <a class="email" href="#">info@ilcsummit2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="sub-section">
+      <div class="container">
+        <h2>ILC Summit 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ILC summit 2023 will be decided through the annual ILC
+          logo competition on May 8, 2023.
+        </p>
+      </div>
+      <hr class="line" />
+      <div class="container">
+        <h2>See the past ILC Summits</h2>
+        <hr />
+        <p>
+          Take a look at the last two ILC Summits which took place in Cape Coast
+          and Accra.
+        </p>
+
+        <div id="summit">
+          <div class="summit-1">
+            <h3 class="summit-header">2021</h3>
+            <p class="summit">ILC Summit 2021 in Cape Coast</p>
+          </div>
+
+          <div class="summit-2">
+            <h3 class="summit-header">2019</h3>
+            <p class="summit">ILC Summit 2019 Accra.</p>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] As I mentioned above, consider breaking this into another section. That will really make your code more cleanly organized :wink: